features/admin_creates_auction.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Admin creates auctions in the admins panel
+  As an admin
+  I should be able to create auctions
+
+  Scenario: Adding an auction
+    Given I am an administrator
+    And there is a tock project to bill to
```

Comment:
I like the idea of only creating it when it is really needed for a test.

The reason for my hesitation is that putting it somewhere general could get us into the territory of creating seed data before each test like C2, whose test suite I ran those morning and OH BOY let me tell you is it slow. 

If you were to make more generic, where would you put it?